x-pack/plugins/enterprise_search/public/applications/app_search/components/create_engine/create_engine_logic.test.ts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { generatePath } from 'react-router-dom';
+
+import { CREATE_ENGINE_SUCCESS_MESSAGE } from './constants';
+import {
+  LogicMounter,
+  mockHttpValues,
+  mockKibanaValues,
+  mockFlashMessageHelpers,
+} from '../../../__mocks__';
+import { ENGINE_PATH } from '../../routes';
+import { formatApiName } from '../../utils/format_api_name';
+import { CreateEngineLogic, DEFAULT_LANGUAGE } from './create_engine_logic';
```

Comment:
[nit] Import order request: https://github.com/elastic/kibana/pull/88494#discussion_r558483164

```
// mocks

// 3rd party imports (e.g. React)

// Shared imports (e.g. helpers, routes)

// Imports directly related to the current component/logic
```
IIRC Jest mocks should typically go first because order does matter for certain kinds of mocks, but I could be wrong on that